examples/userapi/05_conditional_dimension.ipynb

```diff
@@ -0,0 +1,853 @@
+{
```

Comment:
<p>should this cell be moved inside "Use case B" ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/1269/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
never mind, ignore me